cloud_experts_tutorials/rosa-mobb-azure-group-claims.adoc

```diff
@@ -0,0 +1,149 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-azure-group-claims"]
+= Tutorial: Configuring ROSA to use Azure AD group claims
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-azure-group-claims
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-18
+// ---
+// date: '2022-09-19'
+// title: Configure ROSA to use Azure AD Group Claims
+// authors:
+//   - Michael McNeill
+//   - Paul Czarkowski
+// ---
+
+This guide demonstrates how to utilize the OpenID Connect group claim functionality implemented in OpenShift 4.10. This functionality allows an identity provider to provide a user's group membership for use within OpenShift. This guide will walk through the creation of an Azure Active Directory (Azure AD) application, configure the necessary Azure AD groups, and configure Red Hat OpenShift Service on AWS (ROSA) to authenticate and manage authorization by using Azure AD.
+
+This guide will walk through the following steps:
+
+. Register a new application in Azure AD for authentication.
+. Configure the application registration in Azure AD to include optional and group claims in tokens.
+. Configure the OpenShift cluster to use Azure AD as the identity provider.
+. Grant additional permissions to individual groups.
+
+[id="rosa-mobb-azure-group-claims-prerequisites"]
+== Prerequisites
+
+Create a set of security groups and assign users by following link:https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-groups-create-azure-portal[the Microsoft documentation].
```

Comment:
Updated link: https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/how-to-manage-groups